controller.py

```diff
@@ -168,6 +170,22 @@ def register(self, do_get=HTTP.get_with_timeout):
         SHELF_REL = "http://opds-spec.org/shelf"
 
         auth_response = None
+        def get_opds_links(response):
+            type = response.headers.get("Content-Type")
+            if type == "application/opds+json":
```

Comment:
It does, because of `profile`, but right now the only profile is the directory one, and we don't want to import another directory.